packages/core/src/components/modal/modal.tsx

```diff
@@ -24,6 +24,8 @@ export class AtomModal {
   @Prop() alertType?: 'alert' | 'error'
   @Prop() progress?: number
   @Prop() hasFooter?: boolean = true
+  @Prop() disablePrimary?: boolean = false
+  @Prop() disableSecondary?: boolean = false
```

Comment:
suggestion: I hadn't noticed but we can remove it

```suggestion
  @Prop() progress?: number
  @Prop() hasFooter = true
  @Prop() disablePrimary = false
  @Prop() disableSecondary = false
```

Reply:
done